org.gnome.innstereo.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "org.gnome.innstereo",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
```

Comment:
Why not 3.30?

Reply:
Good point. Will upgrade

Reply:
Might be blocked by: https://github.com/flathub/org.freedesktop.Sdk.Extension.gfortran-62/issues/3 ?

Reply:
I don't think so - that just means that it's in the GNOME 3.30 SDK already without needing an extension (although it'll likely move back out again for a future release). note: I have not tested this, it's just the implication of the comments.